bokehjs/src/coffee/models/canvas/cartesian_frame.coffee

```diff
@@ -91,6 +91,10 @@ export class CartesianFrame extends LayoutCanvas
     y_ranges: () -> @_y_ranges
     xscales:  () -> @_xscales
     yscales:  () -> @_yscales
+
+    # These are deprecated and should not be used in new code
+    x_mappers: () -> @_xscales
+    y_mappers: () -> @_yscales
   }
```

Comment:
@mattpap please confirm this is the correct way to do things. The passing test seems to indicate that it is